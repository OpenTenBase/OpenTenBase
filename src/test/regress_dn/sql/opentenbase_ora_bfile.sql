\c regression_ora
-- test bfile
create table test_20231019(c1 int,c2 bfile);
insert into test_20231019 values(1);
/*
execute direct on (coord1) $$select * from test_20231019$$;
execute direct on (coord2) $$select * from test_20231019$$;
*/
drop table test_20231019;

create directory bfiledir as '/tmp';
select DIRECTORY_NAME,NODENAME from pg_directory;
drop directory bfiledir;
select '11111'::bytea::bfile from dual;
select '11111'::clob::bfile from dual;
