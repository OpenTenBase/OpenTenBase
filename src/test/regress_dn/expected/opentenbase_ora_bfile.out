\c regression_ora
-- test bfile
create table test_20231019(c1 int,c2 bfile);
insert into test_20231019 values(1);
/*
execute direct on (coord1) $$select * from test_20231019$$;
execute direct on (coord2) $$select * from test_20231019$$;
*/
drop table test_20231019;
create directory bfiledir as '/tmp';
select DIRECTORY_NAME,NODENAME from pg_directory;
 DIRECTORY_NAME |  NODENAME  
----------------+------------
 BFILEDIR       | datanode_1
(1 row)

drop directory bfiledir;
select '11111'::bytea::bfile from dual;
ERROR:  cannot cast type BYTEA to BFILE
LINE 1: select '11111'::bytea::bfile from dual;
                             ^
select '11111'::clob::bfile from dual;
ERROR:  cannot cast type CLOB to BFILE
LINE 1: select '11111'::clob::bfile from dual;
                            ^
